streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorNode.java

```diff
@@ -185,6 +236,10 @@ public boolean isTerminalNode() {
         return children.isEmpty();
     }
 
+    public void setProcessingExceptionHandler(final ProcessingExceptionHandler processingExceptionHandler) {
+        this.processingExceptionHandler = processingExceptionHandler;
+    }
```

Comment:
I try to avoid adding methods just for test setup. 
Looking at the tests in `ProcessorNodeTest`, I do not see why you need this method. You can equally well pass the processing exception handler into `init()`.

Reply:
Removed. `init()` has been used